Dynatrace/dynatrace-operator
# Separate properties.go into separate files

## Description

`properties.go` was just a dumping ground for helper functions, it's a bit of a mess, this is the first step of cleaning it up.

I did NOT introduce new code (only renamed 1 test, removed some dead code), so if you don't like a name, I probably also do not like it (as sad, it's messy, there are multiple naming styles in it), but didn't want to change every file in the repo.
- We can fix the naming 1 file at a time, (1 PR for each), this should only be the first step 

sidenote: I made this change because I wanted to add properties related to `logModule` as the configuration of the feature is not final(where things will be configured), and wanted to hide that behind properties. (I didn't want to add to the mess)

## How can this be tested?
No functional change
Just check if you like this "new" way of structuring the code
